WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewController.swift

```diff
@@ -7,13 +7,17 @@ class ReaderTabViewController: UIViewController {
     init(viewModel: ReaderTabViewModel) {
         self.viewModel = viewModel
         super.init(nibName: nil, bundle: nil)
-        self.viewModel.filterTapped = { [weak self] (fromView, completion) in
+        viewModel.filterTapped = { [weak self] (fromView, completion) in
             guard let self = self else { return }
             self.viewModel.presentFilter(from: self, sourceView: fromView, completion: { [weak self] title in
                 self?.dismiss(animated: true, completion: nil)
                 completion(title)
             })
         }
+        viewModel.settingsTapped = { [weak self] fromView in
+            guard let self = self else { return }
```

Comment:
The `return` should be on a new line. ([style guide](https://github.com/wordpress-mobile/swift-style-guide))